waiter/src/waiter/marathon.clj

```diff
@@ -63,6 +64,16 @@
         service-id->failed-instances-transient-store max-instances-to-keep service-id failed-instance
         (fn [] #{}) (fn [instances] (-> (scheduler/sort-instances instances) (rest) (set)))))))
 
+(defmacro swallow-out
+  "Like with-out-str, except it swallows anything sent to *out* and returns
+  what the body returns. We wrap this around the calls made to marathonclj
+  that call pprint on the request to avoid the unwanted output on stdout"
+  [& body]
+  `(let [sw# (new StringWriter)
+         value# (binding [*out* sw#] ~@body)]
```

Comment:
I would vote no, since our previous fork we had simply removed the `pprint` and were happy